### PR TITLE
[Github Action] Setup codeception and add test from core. 

### LINKED ISSUE #3: [Github Action] Setup codeception

### PR DESCRIPTION
**Pull request by martineiber**
Resolves #3 

-----

Add test form core bundle. 
Waiting for https://github.com/pimcore/pimcore/pull/14971

### CODE REVIEW

File: .github/ci/files/config/services.yaml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        autoconfigure: true
+
+    #
+    # Notification Services
+    #
+
+    Pimcore\Model\Notification\Service\NotificationService:
+        public: true
+
+    Pimcore\Model\Notification\Service\UserService: ~
```
Comment (dvesh3) [inline .github/ci/files/config/services.yaml]: these can go. see https://github.com/pimcore/admin-ui-classic-bundle/pull/29/files#diff-c47b03734cd2b4337b345eeba955637ba790d51fd98979f6d366d4acbdb104e0L10

File: .github/workflows/codeception.yml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    codeception-tests:
+        name: "Codeception tests"
+        runs-on: "ubuntu-20.04"
+        continue-on-error: ${{ matrix.experimental }}
+        env:
+            PIMCORE_TEST_DB_DSN: "mysql://root@127.0.0.1:33006/pimcore_test?serverVersion=${{ matrix.server_version }}"
+        strategy:
+            matrix:
+                include:
+                    - { php-version: 8.1, database: "mariadb:10.7", server_version: "10.7.7-MariaDB-1:10.7.7+maria~ubu2004", dependencies: highest, experimental: false }
```
Comment (dvesh3) [inline .github/workflows/codeception.yml]:
```suggestion
                    - { php-version: 8.1, database: "mariadb:10.7", server_version: "10.7.7-MariaDB-1:10.7.7+maria~ubu2004", dependencies: lowest, experimental: false }
```
Run at-least one against lowest

File: composer.json [hunk 3]
```diff
@@ -9,15 +9,17 @@
       "*": "dist"
     }
   },
+  "prefer-stable": true,
+  "minimum-stability": "dev",
   "require": {
-    "pimcore/pimcore": "^11.0",
+    "pimcore/pimcore": "^11.x-dev",
```
Comment (dvesh3) [inline composer.json]:
```suggestion
    "pimcore/pimcore": "^11.0",
```
Any specific reason for requiring x-dev?

Comment (martineiber) [reply composer.json]: No, this was just for some testing. Thx for the catch.

File: .github/ci/files/config/services.yaml (hunk 1 above)
Comment (martineiber) [reply .github/ci/files/config/services.yaml]: Why was it removed. it is still used here `src/Controller/Admin/NotificationController.php` right?

Comment (dvesh3) [reply .github/ci/files/config/services.yaml]: because the service class is in the core and admin bundle can still use the service from core.

**Review by dvesh3 — APPROVED** [review]

**Pull request merged**